spec/30-processing-model.md

```diff
@@ -29,8 +29,7 @@ forwards Trace Context headers.
    `parent-id`.
        - If either `trace-id`, `parent-id` or `trace-flags` are invalid, the tracer
       creates a new `traceparent` header and deletes `tracestate`.
-5. The tracer validates the `tracestate` header. If the `tracestate` header
-   cannot be parsed the tracer deletes it.
+5. The tracer _MAY_ validate the `tracestate` header. If the `tracestate` header cannot be parsed the tracer _MAY_ discard the entire header. Invalid `tracestate` entries _MAY_ also be discarded.
```

Comment:
we don't use italics for MUST/MAY

Reply:
The writing edits pr #307 introduces italics for MUST and MAY and I was writing with the assumption that PR would be merged. Do we strictly not want italics (if so, we should probably say so on that PR), or are they ok if we adopt them as a new convention?

Reply:
I would recommend not using italics, the upper-case already makes the words stand out, italics doesn't add anything new and makes plain  text harder to read. I've never seen italics in the ietf RFCs.